.github/workflows/ci_workflow.yml

```diff
@@ -268,6 +268,14 @@ jobs:
       - name: Test
         run: ctest -V
         working-directory: _build
+      - name: Test CMake Consumer
+        if: matrix.build-shared == 'ON'
+        run: |
+          cmake -DCMAKE_PREFIX_PATH=../../../_install .
```

Comment:
I think that `-DCMAKE_BUILD_TYPE=${{ matrix.build-type }} ` is missing; otherwise, it build a release by default.

Reply:
Thanks @hodoulp, good catch!